osgp/protocol-adapter-iec60870/osgp-protocol-adapter-iec60870/src/test/java/org/opensmartgridplatform/adapter/protocol/iec60870/testutils/builders/AsduBuilder.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright 2020 Smart Society Services B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package org.opensmartgridplatform.adapter.protocol.iec60870.testutils.builders;
+
+import org.openmuc.j60870.ASdu;
+import org.openmuc.j60870.ASduType;
+import org.openmuc.j60870.CauseOfTransmission;
+import org.openmuc.j60870.ie.InformationObject;
+
+public class AsduBuilder {
+
+    private final ASduType asduType;
+    private CauseOfTransmission cot = CauseOfTransmission.SPONTANEOUS;
```

Comment:
Why use an abbreviation? I think we generally just take the hit on character count and write things out.

Reply:
Updated